src/ClusterBootstrap/services/monitor/grafana-config-raw/job-status-dashboard.json

```diff
@@ -47,7 +47,7 @@
   "gnetId": null,
   "graphTooltip": 0,
   "id": null,
-  "iteration": 1588881952699,
```

Comment:
What's this?

Reply:
timestamp in epoch millisecond. I guess Grafana uses it in the iteration field.